node/src/Pos/Configuration.hs

```diff
@@ -148,8 +154,17 @@ mdNoBlocksSlotThreshold :: (HasNodeConfiguration, Integral i) => i
 mdNoBlocksSlotThreshold = fromIntegral . ccMdNoBlocksSlotThreshold $ nodeConfiguration
 
 ----------------------------------------------------------------------------
--- Transactions resubmition
+-- Wallet parameters
 ----------------------------------------------------------------------------
 
 pendingTxResubmitionPeriod :: HasNodeConfiguration => Second
 pendingTxResubmitionPeriod = fromIntegral . ccPendingTxResubmissionPeriod $ nodeConfiguration
+
+-- | If 'True', some dangerous endpoints, like one which resets wallet state,
+-- should throw exception if used.
+walletProductionApi :: HasNodeConfiguration => Bool
+walletProductionApi = ccWalletProductionApi $ nodeConfiguration
+
+-- | If 'True', wallet should *not* create new transactions in any way.
```

Comment:
Or re-submit existing pending transactions.

Really it's about not submitting transactions to the network, that's the critical thing.